app/lib/server/startup/email.js

```diff
@@ -288,6 +288,11 @@ settings.addGroup('Email', function() {
 			placeholder: 'email@domain',
 			secret: true,
 		});
+		this.add('Direct_Reply_ReplyTo_Enable', false, {
```

Comment:
As I mentioned on previous comment, this setting should control the `Reply-To` header from regular notifications, so the name of setting is currently misleading..

I'd recommend moving the setting to the `Privacy` section and change its name: 
```suggestion
		this.add('Add_Sender_To_ReplyTo', false, {
```